backend/data/src/main/kotlin/io/tolgee/formats/android/contsants.kt

```diff
@@ -0,0 +1,3 @@
+package io.tolgee.formats.android
+
+const val ANDROID_CDATA_CUSTOM_KEY = "_androidWrapWithCdata"
```

Comment:
I don't like the underscosre

Reply:
What don't you like about it?

Reply:
It's in section described as "Custom attributes specific for some formats", so it's basically the section exactly for this. The "_" looks hacky.

![Snímek obrazovky 2024-03-19 v 15 16 34](https://github.com/tolgee/tolgee-platform/assets/8831144/e9056879-86c0-4ab9-81b0-9ba05df64a07)

Reply:
https://tolgee.io/platform/translation_keys/custom_values#tolgee-internal-custom-values